content/pl/docs/setup/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+no_issue: true
+title: Od czego zacząć
+main_menu: true
+weight: 20
+content_template: templates/concept
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#learning-environment"
+    title: Środowisko do nauki
+  - anchor: "#production-environment"
+    title: Środowisko produkcyjne  
```

Comment:
These anchor names do not match the Markdown:

```markdown
 ## Środowisko do nauki
…
 ## Środowisko produkcyjne
```

maybe this is right:
```yaml
…
 anchors:
  - anchor: "#środowiski-do-nauki"
    title: Środowisko do nauki
  - anchor: "#środowisko-produkcyjne"
    title: Środowisko produkcyjne
```
?

Reply:
I think we should avoid polish special chars in the anchor names. This needs to be verified.

Reply:
Special characters work in anchor names (see eg Korean localization); however, it's fine to use just US-ASCII characters to help with interoperability.

You can also use the English or Polish text in the anchor name, so long as it is all consistent.

Reply:
Fixed.